notebooks/integrations/integration-git.ipynb

```diff
@@ -69,18 +91,46 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "These properties can help easily associate projects and experiments with the exact branches and commits they were run against so we can go back and reference the code later. On the dashboard, experiments can be sorted and filtered by their commit hash."
+    "These properties can help easily associate **projects** and **experiments** with the exact\n",
+    "branches and commits they were run against so we can go back and reference the code later."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " * Running on http://127.0.0.1:8050/ (Press CTRL+C to quit)\n",
+      "127.0.0.1 - - [16/Apr/2021 10:59:23] \"\u001b[37mGET /_alive_bec47650-b0c8-449a-beed-56d8e9dee966 HTTP/1.1\u001b[0m\" 200 -\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Dash app running on http://127.0.0.1:8050/\n"
+     ]
+    }
+   ],
    "source": [
     "from rubicon.ui import Dashboard\n",
     "\n",
-    "Dashboard(\"memory\").run_server()"
+    "\n",
+    "Dashboard(persistence=\"memory\").run_server()"
```

Comment:
nit: the output is tough to read because of the background - we might want to override the css there

Reply:
I just hid all the error divs so that won't happen again